runconfig/parse_test.go

```diff
@@ -322,10 +322,10 @@ func TestParseDevice(t *testing.T) {
 			PathInContainer:   "/something",
 			CgroupPermissions: "rwm",
 		},
-		"/dev/snd:/something:ro": {
+		"/dev/snd:/something:rw": {
 			PathOnHost:        "/dev/snd",
 			PathInContainer:   "/something",
-			CgroupPermissions: "ro",
+			CgroupPermissions: "rw",
 		},
```

Comment:
I think you should also add a test for `/dev/snd:r` (and/or `/dev/snd:rwm`, …). :wink: